cmd/minishift/cmd/start.go

```diff
@@ -270,11 +287,23 @@ func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
 		ShellProxyEnv:    shellProxyEnv,
 	}
 
-	fmt.Printf("Starting local OpenShift cluster using '%s' hypervisor...\n", machineConfig.VMDriver)
+	fmt.Printf(" using '%s' hypervisor ...\n", machineConfig.VMDriver)
 	var hostVm *host.Host
+
+	// configuration with these settings only happen on create
+	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
+	if !isRestart {
+		fmt.Println("-- Instance will be configured with ...")
+		fmt.Println("   Memory:   ", units.HumanSize(float64(machineConfig.Memory*units.MB)))
+		fmt.Println("   vCPUs :   ", machineConfig.CPUs)
```

Comment:
Quite a general term: https://www.google.com/search?q=vcpu as it does not imply affinity.